parlai/scripts/train_model.py

```diff
@@ -81,6 +82,12 @@ def setup_args(parser=None):
     train.add_argument('-lfc', '--load-from-checkpoint',
                        type='bool', default=False,
                        help='load model from checkpoint if available')
+    train.add_argument('-tblog', '--tensorboard-log', type=bool, default=False,
```

Comment:
we can add these from the tensorboardlogger

Reply:
like how we add from models or other things (you extend the args)

Reply:
got it

Reply:
I hadn't seen jason's comment but mine was the same thing